### PR TITLE
Create add consignment status endpoint

### PR DESCRIPTION
**Pull request by techncl**

### CODE REVIEW

File: src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusService.scala [hunk 1]
```diff
@@ -3,17 +3,56 @@ package uk.gov.nationalarchives.tdr.api.service
...
+          consignmentStatusRow.consignmentstatusid,
+          consignmentStatusRow.consignmentid,
+          consignmentStatusRow.statustype,
+          consignmentStatusRow.value,
+          consignmentStatusRow.createddatetime.toZonedDateTime,
+          consignmentStatusRow.modifieddatetime.map(timestamp => timestamp.toZonedDateTime)
+        )
+      }
+    }
+
```
Comment (TomJKing) [inline src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusService.scala:54]: Unnecessary line break

File: src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusServiceSpec.scala [hunk 2]
```diff
@@ -20,7 +20,142 @@ class ConsignmentStatusServiceSpec extends AnyFlatSpec with MockitoSugar with Re
...
+    val expectedStatusValue = "Completed"
+
+    val consignmentStatusRowCaptor: ArgumentCaptor[ConsignmentstatusRow] = ArgumentCaptor.forClass(classOf[ConsignmentstatusRow])
+    val consignmentIdCaptor: ArgumentCaptor[UUID] = ArgumentCaptor.forClass(classOf[UUID])
+
+    val mockGetConsignmentStatusRepoResponse: Future[Seq[ConsignmentstatusRow]] = Future(Seq())
+    val mockAddConsignmentStatusRepoResponse = Future(
+      generateConsignmentStatusRow(expectedConsignmentId, expectedStatusType, expectedStatusValue, None)
+    )
+
```
Comment (TomJKing) [inline src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusServiceSpec.scala:38]: Unnecessary line break

**Review by TomJKing — COMMENTED** [review]
Looks good. Couple of very minor comments. Thanks

File: src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusServiceSpec.scala (hunk 2 above)
Comment (techncl) [reply src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusServiceSpec.scala:38]: This is weird. Here I can see the line break but in the [actual file](https://github.com/nationalarchives/tdr-consignment-api/blob/d305c650cf2cb60af9656c86362653fc8e62a7af/src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusServiceSpec.scala) there isn't one.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**